.gitignore

```diff
@@ -404,3 +404,6 @@ ASALocalRun/
 
 # docker
 *.env
+
```

Comment:
Are those changes needed for this PR?

Reply:
argh, I forgot to remove format changes :( I will remove these from the PR if further changes will be required